pandas/_libs/parsers.pyx

```diff
@@ -215,6 +215,8 @@ cdef extern from "parser/tokenizer.h":
         int64_t header_start        # header row start
         uint64_t header_end         # header row end
 
+        int allow_leading_cols      # Boolean: 1: can infer index col, 0: no index col
```

Comment:
Could you use `bint` here instead if boolean?

Reply:
Done.